--- configure.ac.orig	2022-09-09 21:38:36.534982095 -0400
+++ configure.ac	2022-09-09 21:39:01.563605593 -0400
@@ -1,5 +1,5 @@
 AC_PREREQ([2.69])
-AC_INIT([libgeneral], m4_esyscmd([git rev-list --count HEAD | tr -d '\n']), [tihmstar@gmail.com])
+AC_INIT([libgeneral], [63],, [tihmstar@gmail.com])
 
 AC_CANONICAL_SYSTEM
 AC_CANONICAL_HOST
@@ -9,11 +9,6 @@
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_MACRO_DIRS([m4])
 
-AC_DEFINE([VERSION_COMMIT_COUNT], "m4_esyscmd([git rev-list --count HEAD | tr -d '\n'])", [Git commit count])
-AC_DEFINE([VERSION_COMMIT_SHA], "m4_esyscmd([git rev-parse HEAD | tr -d '\n'])", [Git commit sha])
-AC_SUBST([VERSION_COMMIT_COUNT], ["m4_esyscmd([git rev-list --count HEAD | tr -d '\n'])"])
-AC_SUBST([VERSION_COMMIT_SHA], ["m4_esyscmd([git rev-parse HEAD | tr -d '\n'])"])
-
 # Checks for programs.
 AC_PROG_CXX
 AC_PROG_CC
