--- configure.ac.orig	2022-08-01 09:21:21.762708333 -0400
+++ configure.ac	2022-08-01 09:22:43.212504550 -0400
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.68)
-AC_INIT([libplist], [m4_esyscmd(./git-version-gen $RELEASE_VERSION)], [https://github.com/libimobiledevice/libplist/issues],, [https://libimobiledevice.org])
+AC_INIT([libplist], [2.2.0], [https://github.com/libimobiledevice/libplist/issues],, [https://libimobiledevice.org])
 AM_INIT_AUTOMAKE([dist-bzip2 no-dist-gzip check-news])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES])
 AC_CONFIG_SRCDIR([src/])
@@ -19,11 +19,6 @@
 
 AC_SUBST(LIBPLIST_SO_VERSION)
 
-# Check if we have a version defined
-if test -z $PACKAGE_VERSION; then
-  AC_MSG_ERROR([PACKAGE_VERSION is not defined. Make sure to configure a source tree checked out from git or that .tarball-version is present.])
-fi
-
 # Checks for programs.
 AC_PROG_CC
 AC_PROG_CXX
