--- configure.ac.orig	2022-09-09 22:49:41.346399184 -0400
+++ configure.ac	2022-09-09 22:51:27.231799244 -0400
@@ -1,5 +1,5 @@
 AC_PREREQ([2.69])
-AC_INIT([usbmuxd2], m4_esyscmd([git rev-list --count HEAD | tr -d '\n']), [tihmstar@gmail.com])
+AC_INIT([usbmuxd2], [0],, [tihmstar@gmail.com])
 
 AC_CANONICAL_SYSTEM
 
@@ -11,11 +11,6 @@
 AM_PROG_LIBTOOL
 AM_INIT_AUTOMAKE([subdir-objects])
 
-AC_DEFINE([VERSION_COMMIT_COUNT], "m4_esyscmd([git rev-list --count HEAD | tr -d '\n'])", [Git commit count])
-AC_DEFINE([VERSION_COMMIT_SHA], "m4_esyscmd([git rev-parse HEAD | tr -d '\n'])", [Git commit sha])
-AC_SUBST([VERSION_COMMIT_COUNT], ["m4_esyscmd([git rev-list --count HEAD | tr -d '\n'])"])
-AC_SUBST([VERSION_COMMIT_SHA], ["m4_esyscmd([git rev-parse HEAD | tr -d '\n'])"])
-
 
 # Checks for programs.
 AC_PROG_CXX
